MaxText/layers/quantizations.py

```diff
@@ -606,3 +610,54 @@ def remove_quantized_params(params, aqt_vars):
 
 def configure_kv_quant(config):
   return None if not config.quantize_kvcache else KVQuant(config)
+
+
+def get_basic_config(config, dtype):
+  rules = [
+      qwix.QtRule(
+        module_path='.*',  # Apply to all modules
```

Comment:
For a more exact configuration match, the module_path should be `decoder/{dense,moe}_layers/.+`. I realize that Qwix additionally quantizes the `token_embedder` and the `decoder/logits_dense` module, which are not quantized in AQT.